base/bucket_gocb.go

```diff
@@ -137,14 +137,23 @@ func GetCouchbaseBucketGoCB(spec BucketSpec) (bucket *CouchbaseBucketGoCB, err e
 	spec.MaxNumRetries = 10
 	spec.InitialRetrySleepTimeMS = 5
 
+	// Identify number of nodes to use as a multiplier for MaxConcurrentOps, since gocb maintains one pipeline per data node.
+	// TODO: We don't currently have a process to monitor cluster changes behind a gocb bucket.  When that's available, should
+	//       consider the ability to modify this as the cluster changes.
+	nodeCount := 1
+	mgmtEps := goCBBucket.IoRouter().MgmtEps()
```

Comment:
`mgmtEps` leaves me confused and guessing .. Management Environmental Protection Services is my first guess.  

What do you think about about renaming this to `BucketClusterInfo` or similar to be more descriptive?  The tradeoff is that there is also confusion introduced by differing w/ the upstream API naming .. but I still think this will make our code easier to read.

I guess another approach would be to keep that variable name, but hide/wrap this in a method:

```
func (b gocbbucket) nodeCount() int {
   mgmtEps .. 
   return numNodes (or 1 by default)
}
```

Reply:
I like the gocb naming, as it does a good job differentiating between endpoint types (ViewEp, MgmtEp, N1qlEp, FtsEp).  It's also a useful reminder that all cluster nodes aren't equivalent.

I agree that 'Mgmt' is a bit of an outlier because it covers traditional 'general-purpose' nodes (kv, views, etc), but this feels like a case where we're better off staying consistent with gocb, so that we don't end up introducing confusion between the two names.

Reply:
That's fine, I agree that there's value in being consistent w/ the upstream API.